campusmap/src/info/Info/Info.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { ReactElement } from 'react';
+import { Card, Col, Row, Carousel, Jumbotron, Button, Container, Form } from 'react-bootstrap';
+import { Location } from 'campusmap/src/types';
+
+
+interface Props {
+    location: Location;
+}
+
+/**
+ * Individual Info Result component rendered in a carousel.
+ */
+
+ const Info = ({ location }: Props): ReactElement => {
+    return (
+        <body>
```

Comment:
The body of the page is already created on the main `App` component, so you'll never need to use a `<body>` tag anywhere else